src/testrel.jl

```diff
@@ -470,6 +472,19 @@ function _test_rel_steps(;
     end
 end
 
+function check_flaky(name::String, logs::Vector{LogRecord})
+    retries = 0
+    for log in logs
+        if haskey(log.kwargs, :submit_failed) && log.kwargs[:retry_number] > retries
+            retries = log.kwargs[:retry_number]
+        end
+    end
+
+    if retries > 0
+        @warn "[FLAKY] $name: transaction submission had to be retried $retries times"
```

Comment:
Is it worth somehow noting that the `FLAKY` is not test-specific?

Reply:
We don't really know if it's not test specific. It's probably not in most cases, but there's a weird universe where it could be.